### ISSUE TITLE
Feature request - LURI option for compiled payloads

### ISSUE THREAD

**Issue by mcjon3z** [Veil-Framework/Veil#289]
Currently, it appears that the only payloads which support the meterpreter LURI option are the powershell payloads. Is it feasible to add LURI option for compiled payloads such as c/python/ruby?

**Comment by ChrisTruncer** [comment]
Yeah, that should be possible to get added in. I'll leave this open to note to do that

**Comment by ghost** [comment]
hello i have one error after the installation i 'm on parrotsec
 [!] ERROR #2-3: Can't find the Metasploit Framework (/opt/metasploit-framework/embedded
/framework/).   Run: /usr/share/veil/config/setup.sh --force --silent
and /usr/share/veil/config/setup.sh --force --silent no fix
please how fix this error?
